### ISSUE TITLE
[Bug] Task Bar Icon Pop-up applet does not disappear after selecting option 

### ISSUE THREAD

**Issue by A108384** [BuddiesOfBudgie/budgie-desktop#599]
### Description

When right-clicking on a task bar icon a context menu appears with various options specific to the application.  However, after selecting the desired option (New File, New Windows, Launch New Instance, etc) the pop-up stays in place.  
The pop-up should go away after the user selects the desired option, which is a similar behavior to other context menus.  

### Budgie version

10.9.2

### Operating System

Solus 4.5

### Steps to reproduce the issue

1.  Ensure you an application icon in a Budgie Desktop doc.  (I have my dock positioned at the bottom of my screen)
2. Right click on the application icon
3. Select one of the context menu options.   The selected action does happen, but the context menu remains in place and in-focus.   You must then make a 2nd click on the target (perhaps new) window in order to continue work.     I think the expectation is that the focus would change to the new window or the window you just switched to, etc.    

My typical scenario is to launch a new session (such as an editor session) and i just start typing right after trying to launch a new session, but the window is focused so my next action must be to then click on the new window what was just launched)

### Actual result

![image](https://github.com/user-attachments/assets/cf5211e1-c881-4c71-a0ed-cb47eb5887ba)

### Expected result

When launching a new instance, the expected result is that the new application would launch, that the pop-up would disappear, and that the focus of the cursor (keyboard and mouse pointer) would be to the new instance/window.

**Comment by JoshStrobl** [comment]
I agree that the **desktop actions** invoked in the IconTasklist should hide the popover. The design intent on the _other_ actions (such as for workspace moving, setting always on top) not invoking a hiding of the popover _is_ to enable multiple actions to be performed against a window without having to re-open the popover.
